CHANGES.rst

```diff
@@ -6,6 +6,15 @@ mosaic_pipeline
 
 - Only load patch table when needed. [#1367]
 
+resample
+--------
+
+- Fix an incompatibility with ``numpy 2.0`` in
+  ``resample.resample_utils.build_mask()``. Switched code in
+  ``build_driz_weight()`` to use ``astropy`` equivalent of ``build_mask()``.
+  Deprecated ``resample.resample_utils.build_mask()``[#1383]
+
+
```

Comment:
```suggestion
```

now that #1375 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Fix an incompatibility with ``numpy 2.0`` in ``resample.resample_utils.build_mask()``. Switched code in ``build_driz_weight()`` to use ``astropy`` equivalent of ``build_mask()``. Deprecated ``resample.resample_utils.build_mask()``" > changes/1383.resample.rst
```

(this is just because #1375 was merged while active PRs existed; new PRs will have these instructions in their checklist)

Reply:
Sorry for the confusion! If you'd like I can add a commit here making the change